src/scss/components/_price_variation.scss

```diff
@@ -28,7 +28,7 @@
     }
 
     &[class*='col-'] {
-      @include flex;
+      flex: 1;
```

Comment:
o `flex` da problema no IE, lá está ok?
qual é esse componente?

Reply:
Yes, this change comes from #153. In IE is working fine: 

![image](https://user-images.githubusercontent.com/6332116/36272317-5231e802-1268-11e8-810e-c878400dc31b.png)